data/internal_id.dat

```diff
@@ -0,0 +1,124 @@
+[tech]
```

Comment:
There's no need for the five types to use the same exact range of numbers; that's a potential source of bugs waiting to happen.

Just bump events to 0x02, locations to 0x03, and so on.  We could play at minimizing the ranges, but I feel confident that we're never going to have more than 1.6 million techs to research... so there's no reason to bother.